### PR TITLE
Parse rules

### PR DESCRIPTION
**Pull request by Honny1**
This PR adds the ability to parse rules and show them in an HTML report.

### CODE REVIEW

File: oscap_report/scap_results_parser/scap_results_parser.py [hunk 1]
```diff
@@ -69,5 +69,105 @@ def _debug_show_rules(self):
...
+            fix_dict["system"] = fix.get("system")
+            fix_dict["complexity"] = fix.get("complexity", "")
+            fix_dict["disruption"] = fix.get("disruption", "")
+            fix_dict["strategy"] = fix.get("strategy", "")
+            fix_dict["fix"] = fix.text
+            output.append(Remediation(**fix_dict))
+        return output
+
+    @staticmethod
+    def _get_multi_check(rule):
```
Comment (evgenyz) [inline oscap_report/scap_results_parser/scap_results_parser.py:129]: Any reason it's not Boolean?

File: oscap_report/scap_results_parser/scap_results_parser.py [hunk 2]
```diff
@@ -69,5 +69,105 @@ def _debug_show_rules(self):
...
+        for check in rule.findall(".//xccdf:check", NAMESPACES):
+            if check.get("multi-check") == "true":
```
Comment (evgenyz) [inline oscap_report/scap_results_parser/scap_results_parser.py:131]: It would always be lower-case?

Comment (Honny1) [reply oscap_report/scap_results_parser/scap_results_parser.py:131]: Yes, it should always be lowercase. In [xccdf 1.2](https://csrc.nist.gov/schema/xccdf/1.2/xccdf_1.2.xsd) the `multi-check` is described in lower case and the XSLT transformation handles only `multi-check` with lower case letters.

File: oscap_report/scap_results_parser/scap_results_parser.py (hunk 1 above)
Comment (Honny1) [reply oscap_report/scap_results_parser/scap_results_parser.py:129]: No, there is no reason.

**Review by evgenyz — APPROVED** [review]

**Pull request merged**